src/coreclr/inc/utilcode.h

```diff
@@ -3728,8 +3728,8 @@ extern const CExecutable executable;
 
 void * __cdecl operator new(size_t n, const CExecutable&);
 void * __cdecl operator new[](size_t n, const CExecutable&);
-void * __cdecl operator new(size_t n, const CExecutable&, const NoThrow&);
-void * __cdecl operator new[](size_t n, const CExecutable&, const NoThrow&);
+void * __cdecl operator new(size_t n, const CExecutable&, const std::nothrow_t&);
+void * __cdecl operator new[](size_t n, const CExecutable&, const std::nothrow_t&);
```

Comment:
Do you want `noexcept`? That isn't on the official definition.

Reply:
We could add `noexcept` here for consistency.